File: src/Generated20260401.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260401 {
public:
    // Generated class 20260401_200521_756077
    int value = 614;
    string created = "20260401_200521_756077";
    
    int process() {
        int result = value;
        for (int i = 0; i < 35; i++) {
            result += i * 2;
        }
        return result;
    }
};

int main() {
    Generated20260401 obj;
    cout << obj.process() << endl;
    return 0;
}

File: src/Generated20260401.go
package main

import "fmt"

type Generated20260401 struct {
    // Generated struct 20260401_201104_787206
    Value int
    Created string
}

func (c *Generated20260401) Process() int {
    result := c.Value
    for i := 0; i < 6; i++ {
        result += i * 2
    }
    return result
}

func main() {
    obj := Generated20260401{Value: 644, Created: "20260401_201104_787206"}
    fmt.Println(obj.Process())
}